shared/src/Datadog.Trace.ClrProfiler.Native/util.h

```diff
@@ -25,32 +25,36 @@ const ::shared::WSTRING cfg_log_directory_env = WStr("DD_TRACE_LOG_DIRECTORY");
 // Note that you should also consider adding to the SSI tracer/build/artifacts/requirements.json file
 const shared::WSTRING default_exclude_assemblies[]{
     WStr("aspnet_state.exe"),
+    WStr("collectguestlogs.exe"), // https://github.com/Azure/WindowsVMAgent
     WStr("csc.exe"),
     WStr("dd-trace"),
     WStr("dd-trace.exe"),
     WStr("devenv.exe"),
     WStr("iisexpresstray.exe"),
-    WStr("InetMgr.exe"),
-    WStr("Microsoft.ServiceHub.Controller.exe"),
-    WStr("MSBuild.exe"),
-    WStr("MsDtsSrvr.exe"),
+    WStr("inetmgr.exe"),
+    WStr("microsoft.servicehub.controller.exe"),
+    WStr("msbuild.exe"),
+    WStr("msdtssrvr.exe"),
     WStr("msvsmon.exe"),
-    WStr("PerfWatson2.exe"),
-    WStr("ServiceHub.DataWarehouseHost.exe"),
-    WStr("ServiceHub.Host.CLR.exe"),
-    WStr("ServiceHub.Host.CLR.x86.exe"),
-    WStr("ServiceHub.IdentityHost.exe"),
-    WStr("ServiceHub.RoslynCodeAnalysisService32.exe"),
-    WStr("ServiceHub.SettingsHost.exe"),
-    WStr("ServiceHub.TestWindowStoreHost.exe"),
-    WStr("ServiceHub.ThreadedWaitDialog.exe"),
-    WStr("ServiceHub.VSDetouredHost.exe"),
+    WStr("perfwatson2.exe"),
+    WStr("servicehub.datawarehousehost.exe"),
+    WStr("servicehub.host.clr.exe"),
+    WStr("servicehub.host.clr.x86.exe"),
+    WStr("servicehub.identityhost.exe"),
+    WStr("servicehub.roslyncodeanalysisservice32.exe"),
+    WStr("servicehub.settingshost.exe"),
+    WStr("servicehub.testwindowstorehost.exe"),
+    WStr("servicehub.threadedwaitdialog.exe"),
+    WStr("servicehub.vsdetouredhost.exe"),
     WStr("sqlagent.exe"),
     WStr("sqlbrowser.exe"),
     WStr("sqlservr.exe"),
-    WStr("VBCSCompiler.exe"),
+    WStr("vbcscompiler.exe"),
     WStr("vsdbg"),
     WStr("vsdbg.exe"),
+    WStr("waappagent.exe"),            // https://github.com/Azure/WindowsVMAgent
+    WStr("werfault.exe"),              // WER = Windows Error Reporting - can kick in when a process crashes
+    WStr("windowsazureguestagent.exe") // https://github.com/Azure/WindowsVMAgent
```

Comment:
We should add `powershell.exe`, `pwsh.exe`, and `pwsh` to this list.

Reply:
Agreed